Formula/x11vnc.rb

```diff
@@ -50,7 +50,7 @@ def install
 --- a/x11vnc/solid.c
 +++ b/x11vnc/solid.c
 @@ -177,7 +177,7 @@ unsigned long get_pixel(char *color) {
- 
+
```

Comment:
Careful of accidentally tweaking patch whitespace. It's an excellent way to accidentally break things. (I've been trying to phase out larger inline patches for this very reason).